kansha/services/search/schema.py

```diff
@@ -32,9 +32,9 @@ class MyDoc(Document):
 class FieldType(object):
 
     parent = None
-    name = ''
 
-    def __init__(self, indexed=True, stored=False, default=None):
+    def __init__(self, name='', indexed=True, stored=False, default=None):
```

Comment:
Didn't want to ditch the Declarative Schema where name is provided by attribute assignement.